src/app/users/users-achievements/users-achievements.component.ts

```diff
@@ -67,9 +67,10 @@ export class UsersAchievementsComponent implements OnInit {
   }
 
   initUser(name, planetCode) {
+    const achievementName = name.indexOf('@') > -1 ? name : name + '@' + planetCode;
```

Comment:
Need to handle if the name is null. Cannot read property 'indexOf' of null

Reply:
@yuviii couldn't replicate the case of null name.
Can you please specify the steps or screenshot the indexOf null console error?

Reply:
@yuviii If name is null, this function will not be called (see if statement at line 36).  Could you provide more detail on how you saw that error?